### ISSUE TITLE
cargo raze inside wasm_bindgen/raze of bazelbuild/rules_rust with 0.3.4 has wrong output

### ISSUE THREAD

**Issue by damienmg** [google/cargo-raze#177]
It is not extracting build file for wasm-bindgen-cli in addition to missing a lot of SHA-256.

I have yet to pinpoint the issue...

**Comment by damienmg** [comment]
I get the same issue with 0.3.3, I am unable to install version 0.3.0 and lower due to problem with openssl dependencies on OSX.

**Comment by damienmg** [comment]
Oh I have a hunch, I need to test on linux

**Comment by damienmg** [comment]
Ok on linux that works, except that we are still losing a lot of SHA. I'll keep the bug open for that last issue but that's no longer blocking me.

**Comment by dfreese** [comment]
Is that still with 0.3.4 on linux?

**Comment by damienmg** [comment]
Yes the sha256 issue is with Linux and 0.3.4. I am guessing this is due to
the move to cargo raze

**Comment by damienmg** [comment]
With 0.3.5, https://github.com/damienmg/rules_rust/commit/241a171f87c51bac0e293c2372c30537545b3b7b#diff-e7558b0277a3d8251c3a4f14b46a946c is losing almost all the crates :/ This is not what I expected. I am working around the issue at the moment.

**Comment by UebelAndre** [comment]
Is this related to #188 ?

**Comment by damienmg** [comment]
Yes this is the same issue.
